linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/account_passwords_pam_faillock_audit/bash/shared.sh

```diff
@@ -0,0 +1,23 @@
+# platform = multi_platform_rhel,multi_platform_fedora,multi_platform_ol,multi_platform_rhv,multi_platform_sle
+
+{{{ bash_pam_faillock_enable() }}}
+AUTH_FILES=("/etc/pam.d/system-auth" "/etc/pam.d/password-auth")
```

Comment:
Like in the comment in `shared.yml`, I recommend to use the `bash_pam_faillock_parameter_value` macro.